uiowa/uiowa
# remove uihcsn01

>The decision has been made to de-provision the uihcsn01 application on the Healthcare subscription
https://iowaweb.slack.com/archives/C8SM82ZFZ/p1690303975606049

See https://github.com/uiowa/uiowa/pull/6433/files and https://github.com/uiowa/uiowa/pull/6474/files for reference.

# To Test

- Happy travis build
- String search for uihcsn01 and the uuid b23f6ba2-2103-4497-a2eb-ac95010a3435
- uiowa05 is on svn-45062 so that should stay in travis.yml
